tests/publiccloud/bsc_1205002.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Test to check for bsc#1205002
+#
+# Maintainer: <qa-c@suse.de>
+
+use base "publiccloud::basetest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use publiccloud::ec2;
+use publiccloud::ssh_interactive "select_host_console";
+
+sub run {
+    my $self = shift;
+    select_host_console();    # select console on the host, not the PC instance
+
```

Comment:
because this module suppose to be executed only in EC2 I would as very first line something like 
```perl
die("Not supported provider. This test should run only on EC2") unless is_ec2()
```